charts/kyverno-policies/templates/_helpers.tpl

```diff
@@ -44,3 +44,35 @@ helm.sh/chart: {{ template "kyverno-policies.chart" . }}
 {{- false }}
 {{- end -}}
 {{- end -}}
+
+{{/* Set if a other policies are managed */}}
+{{- define "kyverno-policies.podSecurityOther" -}}
+{{- if has .name .Values.includeOtherPolicies }}
+{{- true }}
+{{- else -}}
+{{- false }}
+{{- end -}}
+{{- end -}}
+
+{{/* Get deployed Kyverno version from Kubernetes */}}
+{{- define "kyverno-policies.kyvernoVersion" -}}
+{{- $version := "" -}}
+{{- with (lookup "apps/v1" "Deployment" .Release.Namespace "kyverno") -}}
```

Comment:
This will break if the user deploys Kyverno into a Namespace other than `kyverno` yes?

Reply:
It uses `.Release.Namespace` so this would not work if the kyverno-policies chart is added to different namespace than Kyverno.  However the version check functions will assume everything is okay if unable to detect the version so if the namespaces don't match, this chart will still install.